src/components/WordGroupSelector/WordGroupSelector.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import { collectedWordGroupsService } from "util/GWUtil/resource";
+import "./WordGroupSelector.css";
+import WordGroupIcon from "./WordGroupIcon";
+import WordSelector from "./WordSelector";
+
+function WordGroupSelector(props) {
+  var collectedWordGroups = null;
+  var wordKeys = null;
+  const [wordGroups, setWordGroups] = useState({});
+
+  useEffect(() => {
+    collectedWordGroupsService.all().then(function(res) {
+      collectedWordGroups = res;
+      wordKeys = Object.keys(collectedWordGroups);
+      setWordGroups({
+        People: collectedWordGroups[wordKeys[0]],
+        Action: collectedWordGroups[wordKeys[1]],
+        Toys: collectedWordGroups[wordKeys[2]],
+        Colors: collectedWordGroups[wordKeys[3]],
+        Animals: collectedWordGroups[wordKeys[4]],
+        Transport: collectedWordGroups[wordKeys[5]],
+        "Body Parts": collectedWordGroups[wordKeys[6]],
+        Clothing: collectedWordGroups[wordKeys[7]],
+        Food: collectedWordGroups[wordKeys[8]],
+        "More Food": collectedWordGroups[wordKeys[9]],
```

Comment:
bothers me a little bit that some of these are strings and some of these aren't LOL